control

```diff
@@ -0,0 +1,15 @@
+Package: ni-custom-device-testing-tools-{labview_version}-support
```

Comment:
Per nipkg convention, if we're adding support to some other product (like LabVIEW) that should be in the package name.
`ni-custom-device-testing-tools-labview-{labview_version}-support`